src/source-napatech.c

```diff
@@ -655,8 +655,9 @@ static PacketQueue packets_to_release[MAX_STREAMS];
  */
 static void NapatechReleasePacket(struct Packet_ *p)
 {
-    PacketFreeOrRelease(p);
```

Comment:
I don't really understand this change. The `ReleasePacket` callback is meant to call `PacketFreeOrRelease` after some cleanup specific to the capture method. Why can't it be done fully here? I mean including `NT_NetRxRelease` here in the callback and getting rid of `packets_to_release` completely?